vcr/issuer/network_publisher.go

```diff
@@ -86,7 +86,7 @@ func (p networkPublisher) PublishCredential(ctx context.Context, verifiableCrede
 
 	payload, _ := json.Marshal(verifiableCredential)
 	tx := network.TransactionTemplate(types.VcDocumentType, payload, key).
-		WithTimestamp(verifiableCredential.IssuanceDate).
+		WithTimestamp(*verifiableCredential.IssuanceDate).
```

Comment:
feels like a panic waiting to happen (although not possible in current flows)

Reply:
True, although this is the network publisher, which ideally is removed at some foreseeable point in future?